crates/re_types/src/components/range1d.rs

```diff
@@ -0,0 +1,282 @@
+// DO NOT EDIT! This file was auto-generated by crates/re_types_builder/src/codegen/rust/api.rs
+// Based on "crates/re_types/definitions/rerun/components/range1d.fbs".
+
+#![allow(trivial_numeric_casts)]
+#![allow(unused_imports)]
+#![allow(unused_parens)]
+#![allow(clippy::clone_on_copy)]
+#![allow(clippy::iter_on_single_items)]
+#![allow(clippy::map_flatten)]
+#![allow(clippy::match_wildcard_for_single_variants)]
+#![allow(clippy::needless_question_mark)]
+#![allow(clippy::new_without_default)]
+#![allow(clippy::redundant_closure)]
+#![allow(clippy::too_many_arguments)]
+#![allow(clippy::too_many_lines)]
+#![allow(clippy::unnecessary_cast)]
+
+use ::re_types_core::external::arrow2;
+use ::re_types_core::ComponentName;
+use ::re_types_core::SerializationResult;
+use ::re_types_core::{ComponentBatch, MaybeOwnedComponentBatch};
+use ::re_types_core::{DeserializationError, DeserializationResult};
+
+/// **Component**: A 1D range, specifying a lower and upper bound.
+#[derive(Clone, Debug, Copy, PartialEq, PartialOrd, bytemuck::Pod, bytemuck::Zeroable)]
+#[repr(transparent)]
+pub struct Range1D(pub [f64; 2usize]);
```

Comment:
it would be nice if this could be a `Range<f64>`. At least I'd expect a conversion from/to it